src/sqlast/query.rs

```diff
@@ -197,6 +192,23 @@ impl ToString for SQLSelectItem {
     }
 }
 
+/// A table reference is one table factor followed by any number of joins.
```

Comment:
I get that this follows the names of the productions in ANSI SQL syntax, but maybe `TableWithJoins`?

Reply:
Yeah, I find them hard to remember too. I've taken your suggestion for `TableWithJoins`.